### DISCUSSION TITLE
Como se comportar nos primeiros dias de trabalho alocado em um ambiente junto ao time?

### DISCUSSION THREAD

**Discussion by thiagosantoscunha** [frontendbr/forum#1188] [Discussão]
Essa semana fui contratado para trabalhar alocado em uma empresa de desenvolvimento, onde irei atuar como Desenvolvedor Pleno, especificamente na área de frontend com Angular 2+.

Desde quando fui demitido de forma obrigada em uma agência de publicidade há 5 anos atrás, não trabalho em equipe, e geralmente, era somente eu trabalhando com desenvolvimento. Desde quando sai dessa empresa, sempre trabalhei de forma remota.

Gostaria de saber como devo me comportar para readmitir-me em um time de desenvolvimento?
Saber que perguntas devo fazer, o que nunca fazer, cultura, dentre outros fatores.

**Comment by ViniciusGularte** [comment]
Acho que depende da cultura da empresa que você ira atuar... já atuei em dois times  diferentes e mudou bastante coisa de um para outro,na primeira semana acredito melhor observar mais como funciona tudo para tentar acompanhar a roda do local e a cultura que eles tem lá dentro. 

Quando trabalhamos sozinhos e vamos trabalhar com uma equipe é um pouco mais difícil se habituar, eu passei por isso, eu sempre digo, é mais fácil lidar com códigos que com pessoas. 

Procurar conhecer os colegas de trabalho, cada um tem suas manias , e ir aos poucos se habituando ao ambiente .Não acho muito bom chegar de cara falando o jeito X e Y para se fazer as coisas, deixa eles conheceram teu trabalho e você conhecer o deles e aos poucos vai exercendo influencia e sendo 
influenciado pelo time.

Boa sorte no novo emprego :smile:

**Comment by thiagosantoscunha** [comment]
@ViniciusGularteMartin Perfeita colocação... muito obrigado pela contribuição!
